src/OpenSage.Game/Gui/Wnd/WndInputMessageHandler.cs

```diff
@@ -76,9 +76,10 @@ bool GetControlAtPoint(in Point2D mousePosition, out Control control, out Point2
                                 new WndWindowMessage(WndWindowMessageType.MouseMove, control, mousePosition),
                                 context);
                         }
-                        return mouseOverControls.Count > 0
+                        /*return mouseOverControls.Count > 0
```

Comment:
I think that change does cause the mouse to click "through" controls? Why was this necessary?

Reply:
No, controls I checked works normally. What that change do is prevent a mouse hover stealing by wnd controls. If there is some control that broken by this change, it should be treated specifically.